lib/rubber/cloud/fog.rb

```diff
@@ -63,8 +63,8 @@ def start_instance(instance)
         compute_provider.servers.get(instance.instance_id).start
       end
 
-      def create_static_ip
-        address = compute_provider.addresses.create
+      def create_static_ip(domain = 'standard')
+        address = compute_provider.addresses.create(domain: domain)
```

Comment:
Is this specific to AWS? Would we want to put it in the AWS base class instead?

Reply:
To be honest I'm not sure about that but from what I understands the whole Static IP concept is specific to AWS so it really doesn't matter - if so everything related to Static IPs should be moved to AWS base class.